### ISSUE TITLE
Google TV w/ Chromecast (2020) - AppCleaner locks up desipite accessibility service turned on in Settings

### ISSUE THREAD

**Issue by mcl21014** [d4rken/sdmaid-public#4149]
Device: Google TV w/ Chromecast (2020)
SDM version: 5.0.0 beta (sideloaded)
Root status: unrooted
SDM Unlocker version: 4.4.1
Install ID: 36e8dae7-7217-432b-9036-4487663ff591

What happened?
User manually enables Accessibility service not within SDM, but under Settings on Android TV. Open SDM and run the AppCleaner with the service toggled on under Settings. Now, the AppCleaner will start but hang and not clean even the first entry. I had to power off the device as I couldn't stop SD Maid.
What did you expect to happen?
AppCleaner should run as normal without hanging on applications.
What actions did you take?
Run AppCleaner with the Accessibiility service manually turned on.
Can you reproduce it, i.e. does this happen everytime?
Yes.

Logcat: 
[sdmaid_logfile_1603303805724.txt](https://github.com/d4rken/sdmaid-public/files/5417996/sdmaid_logfile_1603303805724.txt)

**Comment by d4rken** [comment]
Seems to be related to SD Maid not recognizing the system settings app window.

```
D/ACC:ACCrawler: Looking for window root (intent=Intent { act=android.settings.APPLICATION_DETAILS_SETTINGS dat=package:com.android.vending flg=0x10818000 }).
```

```
V/ACC:Service: onAccessibilityEvent(event=EventType: TYPE_WINDOW_CONTENT_CHANGED; EventTime: 13395848; PackageName: com.android.tv.settings;
```

**Comment by d4rken** [comment]
Might be able to add a workaround, just need to detect if it is an Android TV type device.

**Comment by d4rken** [comment]
I've added preliminary support in v5.0.8.

**Comment by Tbo29** [comment]
Hallo, just tested it on my Chromecast Google TV. Yes, it cleans now. Opens every app page to be cleaned and deletes the cache, albeit very slowly, but the important thing is, it works. Refuses to clean the cache of the Google Play Store.  It opens the page of it, but no cleaning. Is this intentionally?

**Comment by d4rken** [comment]
There will be a fix in 5.0.9, SD Maid needs to scroll to find the clear cache option.
